ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/slider/SliderScreen.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.android.wearable.composeadvanced.presentation.ui.slider
```

Comment:
Nit: I think it's fine to move both slider and stepper from own packages into the `com.example.android.wearable.composeadvanced.presentation.ui.valuedisplay` and perhaps rename it to `com.example.android.wearable.composeadvanced.presentation.ui.userinput`